Mayhemfile

```diff
@@ -0,0 +1,10 @@
+# Namespaced project name that the target belongs to
+project: webrtc-sdp
+# Target name (should be unique within the project)
+target: fuzz_target_parse_sdp
+# Base image to run the binary in.
+image: ghcr.io/tleong073/fuzz_target_parse_sdp
```

Comment:
You don't need to specify the image here, since it is provided as part of the action (see https://github.com/mayhemheroes/webrtc-sdp/pull/2/files#diff-04bf10df0265e24171447bef0fc3b679cced786fefb86ae3b14af9950d41de83R52). Otherwise, looks good to me!